sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/StrictTlsContext.java

```diff
@@ -10,7 +10,7 @@
 /**
  * Context that removes SSLv2Hello protocol from every SSLEngine created.
  */
-public class StrictTlsContext extends SSLContext {
+class StrictTlsContext extends SSLContext {
```

Comment:
Any reason for reducing the visibility of this class?

Reply:
Because it was only used in this package. I had accidentally copied over public. I like to decrease visibility and add it if we really need it.